lib/tasks/access_tokens.rake

```diff
@@ -0,0 +1,13 @@
+namespace :access_tokens do
+  desc "Remove an access token for a user"
+  task :remove_access_token, %i[owner_email] => :environment do |_, args|
```

Comment:
This should probably be just `owner` instead of `owner_email`, as we don't normally put the email address in the owner field.

```suggestion
  task :remove_access_token, %i[owner] => :environment do |_, args|
```

Reply:
Thanks @lfdebrux. What format do the values for `owner` usually take? I had assumed as it's an API token it'd be sensible to make it email addresses...

Reply:
At the moment we don't have a consistent format, in the production database we've used `kelvin.gan`, `kelvin_gan` and `kelvin-gan`, but the most common is the kebab case (`kelvin-gan`).

Reply:
Right well I'm going to ask from this day on we use email addresses :)